pkg/event/event_tbc.go

```diff
@@ -361,8 +361,9 @@ func (e *EventHandler) inSyncCondition(cfgName string) bool {
 		glog.Info("Leading clock in-sync condition is pending initialization")
 		return false
 	}
-	worstDpllOffset := e.getLargestOffset(cfgName)
-	if math.Abs(float64(worstDpllOffset)) < float64(e.LeadingClockData.inSyncConditionThreshold) {
+
+	meanOffset := e.getMeanOffset(cfgName)
+	if math.Abs(meanOffset) < float64(e.LeadingClockData.inSyncConditionThreshold) {
```

Comment:
Don't we omit all other offsets in the system from the decision logic when looking only at the leading DPLL's one? I am afraid of the situation when the leading DPLL offset is fine, but another DPLL or ts2phc offset is out of bounds, but we still declare "in sync".